core/src/main/java/org/apache/carbondata/core/cache/dictionary/ReverseDictionaryCache.java

```diff
@@ -43,7 +43,7 @@
    * Attribute for Carbon LOGGER
    */
   private static final LogService LOGGER =
-      LogServiceFactory.getLogService(ForwardDictionaryCache.class.getName());
+      LogServiceFactory.getLogService(ReverseDictionaryCache.class.getName());
```

Comment:
i suggest that don't need to change this part in this PR.  it would be better that each pr clearly fix each issue.